app/models/comment.rb

```diff
@@ -1,4 +1,8 @@
 class Comment < ApplicationRecord
   belongs_to :user, class_name: 'User', foreign_key: 'author_id'
   belongs_to :post, class_name: 'Post', foreign_key: 'post_id', counter_cache: true
+
```

Comment:
- [ ] According to the ERD, `:user` shouldn't be the target for the association.  It should be `author`.  
- [ ] Also, `:post` does not need a class name because the name of the class is the same.
This is a simple way to do it, and meets all requirements:

```ruby
belongs_to :author, class_name: 'User'
belongs_to :post
```